src/mongocxx/collection.hpp

```diff
@@ -485,6 +486,23 @@ class MONGOCXX_API collection {
     ///
     void rename(stdx::string_view new_name, bool drop_target_before_rename = false);
 
+    ///
+    /// Sets the read_concern for this collection. Changes will not have any effect on existing
+    /// cursors or other read operations which use the previously-set read concern.
+    ///
+    /// @param rc
+    ///   The new @c read_concern
+    ///
+    /// TODO add link to docs once they exist.
+    ///
+    void read_concern(class read_concern rc);
+
+    ///
+    /// Gets the read_concern for the collection.
```

Comment:
Can you add a comment here that mentions CXX-758, so we remember to change this once that is fixed?